packages/cli/src/databases/migrations/common/1690000000040-AddMfaColumns.ts

```diff
@@ -0,0 +1,35 @@
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+import { TableColumn } from 'typeorm';
+
+export class AddMfaColumns1690000000030 implements ReversibleMigration {
+	async up({ queryRunner, tablePrefix }: MigrationContext) {
+		await queryRunner.addColumns(`${tablePrefix}user`, [
```

Comment:
adding removing columns like causes data-loss because of a typeorm issue. Fix [here](https://github.com/n8n-io/n8n/pull/7006).